models/userRole.js

```diff
@@ -0,0 +1,29 @@
+module.exports = (sequelize, DataTypes) => {
+  const UserRole = sequelize.define('UserRole', {
+    // id: {
+    //   type: DataTypes.INTEGER,
+    //   primaryKey: true,
+    //   allowNull: false
+    // },
+    name: DataTypes.STRING,
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      references: {
+        model: 'Users',
+        key: 'id'
+      }
+    }
+  }, {
+    classMethods: {
+      associate: (models) => {
+        UserRole.belongsTo(models.User, { foreignKey: 'userId' })
+      }
+    },
+    instanceMethods: {
+
```

Comment:
You need to create another migration with the `Role` table, and the role contains the name, and then on this migration, we reference the `roleId` 

The way you implemented we will have a new role for each user, and that's not what we want, we want to have defined roles and reference the user to that role.

But now if you give a `Role` table, it will affect the Member Roles table, so we need another name.

We can call `UserTypes` with a table `Types`, or `UserViews` with a table `Views`

Or we rename the Roles for Members.